packages/components/pie-card-container/src/card-container.scss

```diff
@@ -1 +1,95 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+@mixin card-interactive-states($bg-color) {
+    &:hover:not([disabled]) {
+        --hover-modifier: calc(-1 * var(--dt-color-hover-01));
+        --card-bg-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), calc(var(#{$bg-color}-l) + var(--hover-modifier)));
+    }
+
+    &:active:not([disabled]) {
+        --active-modifier: calc(-1 * var(--dt-color-active-01));
+        --card-bg-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), calc(var(#{$bg-color}-l) + var(--active-modifier)));
+    }
+}
+
+// By default, the card is styled to the 'Default' variant designs
+.c-card-container {
+    --card-bg-color: var(--dt-color-container-default);
+    --card-radius: var(--dt-radius-rounded-c);
+    --card-shadow: var(--dt-elevation-01);
+
+    display: block;
+    color: currentcolor;
+    background-color: var(--card-bg-color);
+    border-radius: var(--card-radius);
+    text-decoration: none;
+    position: relative;
+
+    @include card-interactive-states('--dt-color-container-default');
+
+    &[disabled] {
+        --card-bg-color: var(--dt-color-disabled-01);
+        pointer-events: none;
+    }
+
+    // We should not apply the default elevation styling if the card is focused
+    // but we want to make sure that it is still applied when the card is clicked
+    &:not(:focus-within),
+    &:active:focus-within {
+        box-shadow: var(--card-shadow);
+    }
+
+    // We want to stretch the <a> tag over the entire card to make the entire component interactive
+    > a {
```

Comment:
would these styles get applied if the card itself isn't a link, but part of its content has a link? like
<div>
fancy card, <a>Learn more about it</a>
</div>

Reply:
Nope :) because it is only selecting the immediate child anchor

Reply:
@raoufswe that said, I do not believe the example you've created is one the card designs currently support, as they all require the entire card to be clickable, rather than nested content

Reply:
@raoufswe I don't think those will be cards, probably worth catching up with Naz as I am just quoting the figma designs :)

Reply:
@raoufswe I will create a ticket to look into this and send to @kevinrodrigues